js/bitmex.js

```diff
@@ -228,6 +237,133 @@ module.exports = class bitmex extends Exchange {
+        //                    "maxFee": "10000000"
+        //                }
+        //            ]
+        //        },
+        //     }
+        //
+        const result = {};
+        const keys = Object.keys (response);
+        for (let i = 0; i < keys.length; i++) {
+            const key = keys[i];
+            const currency = response[key];
+            const id = this.safeString (currency, 'asset');
+            const code = this.safeCurrencyCode (id);
+            const name = this.safeString (currency, 'name');
+            const chains = this.safeValue (currency, 'networks', []);
+            let depositEnabled = false;
+            let withdrawEnabled = false;
+            const networks = {};
+            for (let j = 0; j < chains.length; j++) {
+                const chain = chains[j];
+                const networkId = this.safeString (chain, 'asset');
+                const network = this.safeNetwork (networkId);
+                const isDepositEnabled = this.safeValue (chain, 'depositEnabled', false);
+                const isWithdrawEnabled = this.safeValue (chain, 'withdrawalEnabled', false);
+                const active = (isDepositEnabled && isWithdrawEnabled);
+                if (isDepositEnabled) {
+                    depositEnabled = true;
+                }
+                if (isWithdrawEnabled) {
+                    withdrawEnabled = true;
+                }
+                networks[network] = {
+                    'info': chain,
+                    'id': networkId,
+                    'network': network,
+                    'active': active,
+                    'deposit': isDepositEnabled,
+                    'withdraw': isWithdrawEnabled,
+                    'fee': this.safeNumber (chain, 'withdrawalFee'),
+                    'precision': undefined,
+                    'limits': {
+                        'withdraw': {
+                            'min': undefined,
+                            'max': undefined,
+                        },
+                        'deposit': {
+                            'min': undefined,
+                            'max': undefined,
+                        },
+                    },
+                };
+            }
+            const currencyEnabled = this.safeValue (currency, 'enabled');
+            const currencyActive = currencyEnabled || (depositEnabled || withdrawEnabled);
+            const precision = this.parseNumber (this.parsePrecision (this.safeString (currency, 'scale')));
+            this.options['tempCurrencyPrecision'][code] = precision;
+            result[code] = {
+                'id': id,
+                'code': code,
+                'info': currency,
+                'name': name,
+                'active': currencyActive,
+                'deposit': depositEnabled,
+                'withdraw': withdrawEnabled,
+                'fee': undefined,
+                'precision': precision,
+                'limits': {
```

Comment:
this value seems a bit off 😅 min btc deposit: 10000 BTC
![image](https://user-images.githubusercontent.com/43336371/197791158-d95e10b3-965f-4e55-b326-1c9467727748.png)

Reply:
uh oh. nice catch. I think I will include that to get tested in `test.currency.js`.

Reply:
I've manually checked and now the current fix correctly calculates and matches the UI amounts shown in deposit/withdraw pages.